packages/lodestar-beacon-state-transition/test/unit/stateTransition/block/operations/deposit2.test.ts

```diff
@@ -57,15 +58,16 @@ it("should verify deposit 2222", async () => {
   `;
     
   const deposit = config.types.Deposit.fromJson(JSON.parse(specJson));
-  const privKey: Uint8Array = new Uint8Array(32);
-  privKey[31] = 1;
+  const privKey: Uint8Array = PrivateKey.fromInt(1).toBytes();
   // Make sure we use correct privKey
   assert.deepEqual(deposit.data.pubkey, generatePublicKey(privKey));
   const domain = computeDomain(DomainType.DEPOSIT, config.params.GENESIS_FORK_VERSION);
   const signingRoot = computeSigningRoot(config, config.types.DepositMessage, deposit.data, domain);
   const signature = bls.sign(privKey, signingRoot);
-  // This is different from the signature we saw above
+  // The signature above is from spec 0.10.1: https://media.githubusercontent.com/media/ethereum/eth2.0-spec-tests/v0.10.1/tests/minimal/phase0/genesis/initialization/pyspec_tests/initialize_beacon_state_from_eth1/deposits_0.yaml
+  // This is the same to spec 0.10.0: https://media.githubusercontent.com/media/ethereum/eth2.0-spec-tests/v0.10.0/tests/minimal/phase0/genesis/initialization/pyspec_tests/initialize_beacon_state_from_eth1/deposits_0.yaml
```

Comment:
Signature that we generate matches 0.10.0 spec?
Did you copy their signature from spec tests repo because you should copy one from tar.gz arhive since they didn't update test files in repo

We are passing 0.10.1 signing spec tests so it shouldn't be bls

Reply:
> Signature that we generate matches 0.10.0 spec?
Yes, it matches the value specified in the provided link.
> Did you copy their signature from spec tests repo because you should copy one from tar.gz arhive since they didn't update test files in repo
No, I used `config.types.Deposit.toJson(deposit).toJson()` in spec test and `console.log` from there, then use the same value in this unit test by copying the string.

I have no ideas for now, don't see anything in v0.10.1 that may cause the signature change